macros/models/_sector/tokens/balances_daily.sql

```diff
@@ -0,0 +1,23 @@
+{%- macro balances_daily(balances_base) %}
+select
+    *,
+    {{ dbt_utils.generate_surrogate_key(['day', 'type', 'address', 'contract_address', 'token_id']) }} as unique_key
```

Comment:
This line handles the `null` value in unique_key.

Reply:
just validating you are stating the use case for this surrogate key, not asking a question?